src/library/dlgtagfetcher.cpp

```diff
@@ -371,19 +386,13 @@ void DlgTagFetcher::apply() {
         m_pWorker->start();
     }
 
-    statusMessage->setText(tr("Selected metadata applied"));
-
-    m_pTrack->replaceMetadataFromSource(
-            std::move(trackMetadata),
-            // Prevent re-import of outdated metadata from file tags
-            // by explicitly setting the synchronization time stamp
-            // to the current time.
-            QDateTime::currentDateTimeUtc());
+    statusMessage->setText(tr("Selected cover art applied"));
 }
```

Comment:
IIUC, since the metadata is not replaced anymore, this will cause the metadata to not be applied if people press "Apply Cover" and then exit the dialog, right?

Reply:
Yup, correct.

Reply:
Right, that seems unexpected to me. If I pressed the Apply button and exited the dialog, I'd expect the changes to be applied.

Reply:
Ah, nevermind, I found the issue. I was misunderstood the code.